docs/ops.html

```diff
@@ -1604,6 +1604,11 @@ <h4 class="anchor-heading"><a id="remote_jmx" class="anchor-link"></a><a href="#
         <td>kafka.server:type=ReplicaManager,name=AtMinIsrPartitionCount</td>
         <td>0</td>
       </tr>
+      <tr>
+        <td>Producer Id counts</td>
+        <td>kafka.server:type=ReplicaManager,name=ProducerIdCount</td>
+        <td>Count of all producer ids for each replica on the broker.</td>
```

Comment:
From the KIP, this should probable be:
> The total number of active transactional / idempotent producer ids in all partitions maintained in the broker.

Reply:
I think what is written here is fine. If we want to change the wording to "in" instead of "for" that may slightly improve the meaning. 

Active here is a bit subjective, since producer IDs may be inactive but not expired yet :)

Reply:
My only concern is that non-expert user may not understand that producer IDs are not generated for all producers. It would be nice if we can explicitly clarify here that this metrics only counts the producers that create a producer ID i.e. idempotent/transactional.

Reply:
I guess if we want to be really pedantic -- the producer IDs here are for idempotent producers, its just that transactions require idempotent producers. Transactional IDs are not stored here, so the original comment could also be a bit confusing. 

If we want to say transactional and idempotent producers create producer IDs after the original sentence we can do that.

Reply:
That would be helpful. Thank you :)